survey/src/main/kotlin/com/surveyor/survey/services/userService.kt

```diff
@@ -0,0 +1,15 @@
+package com.surveyor.survey.services
+
+import com.surveyor.survey.repository.UserRepository
+import com.surveyor.survey.entities.User
+import org.springframework.stereotype.Service
+import java.util.Optional
+
+@Service
+class UserService(private val userRepo: UserRepository) {
+
+    fun findByEmail(email: String): Optional<User> {
```

Comment:
Why using `Optional` If we use kotlin? why not return it as following.

```suggestion
    fun findByEmail(email: String): User? {
```